src/components/widgets/Map.vue

```diff
@@ -852,6 +860,21 @@ const setHomePosition = async (homePosition: [number, number]): Promise<void> =>
 
 // Allow executing missions
 const executeMissionOnVehicle = async (): Promise<void> => {
+  if (missionStore.showChecklistBeforeArm === true) {
+    isMissionChecklistOpen.value = true
+    return
+  }
+
+  try {
+    await vehicleStore.startMission()
+  } catch (error) {
+    openSnackbar({ message: 'Failed to start mission.', variant: 'error' })
+  }
+  return
+}
```

Comment:
This code is duplicating the one inside the `handleMissionChecklistConfirmed` method. Better to just do a `handleMissionChecklistConfirmed(true)` here.

Ideally, so it doesn't get out of context, the method should be renamed to "tryToStartMission", without any arguments, and in the `@confirmed` directive just use the following:

```
    @confirmed="tryToStartMission"
```

as the @confirm will only be executed when the user clicks to go.